spyder/plugins/editor/plugin.py

```diff
@@ -2701,6 +2706,35 @@ def re_run_last_cell(self):
         editorstack = self.get_current_editorstack()
         editorstack.re_run_last_cell()
 
+    @Slot(int)
+    def save_bookmark(self, num):
+        """Save current cursor position as bookmark in slot based
+        on number of shortcut e.g Ctrl+Shift+1 for slot 1"""
+        filename = self.get_current_filename()
+        position = self.get_current_editor().get_position('cursor')
+        self.bookmarks[num] = (filename, position)
+        CONF.set('editor', 'bookmarks', self.bookmarks)
+
+    @Slot(int)
+    def load_bookmark(self, num):
+        """Set cursor to saved file and position in selected slot
+        based on number of shortcut e.g Ctrl+Alt+1 for slot 1"""
+        filename, position = self.bookmarks[num]
+        if not osp.isfile(filename):
+            self.last_edit_cursor_pos = None
+            return
+        else:
+            self.load(filename)
+            editor = self.get_current_editor()
+            if position < editor.document().characterCount():
+                editor.set_cursor_position(position)
+
+    def init_bookmarks(self):
+        """Load last saved cursor bookmarks for config file"""

```

Comment:
from config file, presumably. Also, period at the end (per PEP 257, docstrings should always end with a period).